en/streaming-integrator/docs/setup/deploying-si-as-minimum-ha-cluster.md

```diff
@@ -26,7 +26,7 @@ events to active node. To achieve this, you can use a load balancing mechanism t
 
 In order to configure a minimum HA cluster, the following prerequisites must be completed:
 
-  - It is recommended to run this setup with two CPUs. Each CPU should have four cores, and 4GB memory.
+  - To run this set up, you need a CPU with four cores, and a total memory of 4GB.
```

Comment:
Can we mention that this is for one SP instance

Reply:
Changed as specified.